ece_core/18614.md

```diff
@@ -0,0 +1,19 @@
+# 18-614: Microelectromechanical Systems (MEMS)
+
+## Course Overview
+
+This course covers fabrication and design fundamentals for Microelectromechanical Systems at the micro scale. Expect to learn basic principles from microstructure fabrication, mechanics of silicon and thin-film materials, electrostatic force, capacitive motion detection, fluidic damping, piezoelectricity, piezoresistivity, and thermal micromechanics. 
+
+## What to expect
+
+- **Homework:** Weekly assignments based on the lecture and sometimes examples from the slides provided. Make sure to show all work and reference formulas step by step to maximize your grade.
+- **Exam:** Expect at least one exam as the final, but it has changed year by year and some previous classes had a midterm. Extremely doable, but only if you understand the lectures and make use of the practice exams and homework provided. Also be mindful of time management on the exam as it is a huge factor and at times people may not be able to finish in time.
```

Comment:
Why is the exam hard if people don't finish on time?

What is the format of the exam? Is it more computational based or more short answer? etc.